app/services/open_library_service.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class OpenLibraryService
+  def book_info(isbn)
+    url = "https://openlibrary.org/api/books?format=json&jscmd=data&bibkeys=ISBN:#{isbn}"
```

Comment:
Move this to bare uri of Httparty

Reply:
Excuse me, Nico, i don't understand this

Reply:
https://www.rubydoc.info/github/jnunemaker/httparty/HTTParty%2FClassMethods:base_uri